models/articles.models.js

```diff
@@ -22,3 +22,17 @@ exports.selectArticleById = (article_id) => {
     return rows[0];
   });
 };
+
+exports.selectAllCommentsByArticleId = (article_id) => {
+  return Promise.all([
```

Comment:
Great use of promise.all here to differentiate between a valid empty response and an article that doesn't exist!